spec/unit/foreman_smartproxy_rest_v3_spec.rb

```diff
@@ -22,6 +24,9 @@
   describe '#create' do
     it 'sends POST request' do
       provider.expects(:request).with(:post, 'api/v2/smart_proxies', {}, is_a(String)).returns(mock(:code => '201'))
```

Comment:
It'd be really great to be able to verify the JSON we're generating instead of `is_a(String)` but I couldn't find a good way to do this - any ideas?